src/cf-typography.less

```diff
@@ -1171,6 +1178,7 @@
 */
 
 .list__branded {
+    /* TODO: Why do we need .list_item when li will always be in the list */
```

Comment:
Yeah, I don't know. `.list_item` seems unnecessary here. This whole pattern can go away though, since we decided not to ever use green bullets.